users/tamashia/index.md

```diff
@@ -55,7 +55,7 @@ Under Construction 8/4/17
 ***
 
 ### Publications
-1. R.O. Hughes, J.T. Burke, R.J. Casperson, S. Ota, 1, S. Fisher, J. Parker, C.W. Beausang, M. Dag, P. Humby, J. Koglin, E. McCleskey, A.B. McIntosh, A. Saastamoinen, A.S. Tamashiro, E. Wilson, T.C. Wu, "The hyperion particle-&gamma detector array", *Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment*, **856**, 47-52 (2017).
+1. R.O. Hughes, J.T. Burke, R.J. Casperson, S. Ota, 1, S. Fisher, J. Parker, C.W. Beausang, M. Dag, P. Humby, J. Koglin, E. McCleskey, A.B. McIntosh, A. Saastamoinen, A.S. Tamashiro, E. Wilson, T.C. Wu, "The hyperion particle-\gamma detector array", *Nuclear Instruments and Methods in Physics Research Section A: Accelerators, Spectrometers, Detectors and Associated Equipment*, **856**, 47-52 (2017).
```

Comment:
@Aarontama if you want to use tex stuff you will need to put it in between $$ <insert> $$

At least that _should_ make it work... I've had troubles with this in the past... @rtrp/admins

Reply:
@Aarontama @albeanth 

Inline TeX is $$ *argument* $$